apps/client/src/shared/apis/carousel/carousel-queries.ts

```diff
@@ -1,9 +1,11 @@
 import { queryOptions } from '@tanstack/react-query';
 import { getTicketing } from './ticketing';
+import { getLatestPerformances } from './performances';
 
 export const CAROUSEL_QUERY_KEY = {
   ALL: ['carousel'],
   TICKETING: ['carousel', 'ticketing'],
+  LATEST_PERFORMANCES: ['carousel', 'latestPerformances'],
```

Comment:
```suggestion
export const CAROUSEL_QUERY_KEY = {
  ALL: ['carousel'],
  TICKETING: [...CAROUSEL_QUERY_KEY.ALL, 'ticketing'],
  LATEST_PERFORMANCES: [...CAROUSEL_QUERY_KEY.ALL, 'latestPerformances'],
```
이렇게 수정하면 쿼리키 늘어났을때 관리가 더 쉬울 것 같습니다~!